packages/shared/components/organisms/ReceiveAddressSection.svelte

```diff
@@ -0,0 +1,83 @@
+<script lang="typescript">
+    import { appSettings } from '@core/app'
+    import { QR, Text } from 'shared/components'
+    import { AddressBox, OnboardingButton } from 'shared/components/atoms'
+    import { localize } from '@core/i18n'
+    import { selectedAccount } from '@core/account'
+    import { FontWeightText } from 'shared/components/Text.svelte'
+    import { openPopup } from '@lib/popup'
+
+    export let addressFontSize = 'base'
+
+    let addressBoxElement: AddressBox
+
+    $: recieveAddress = $selectedAccount.meta.publicAddresses[0].address
+    $: darkModeEnabled = $appSettings.darkMode
+
+    function handleSendClick() {
+        openPopup({
+            type: 'sendForm',
+            overflow: true,
+        })
+    }
+
+    function handleRecieveClick() {
+        addressBoxElement.copyAddress()
+    }
+</script>
+
+<receive-details class="flex flex-col space-y-6 justify-between cursor-pointer">
```

Comment:
```suggestion
<receive-details class="flex flex-col space-y-6 justify-between">
```

Remove `cursor-pointer` style from the entire component because it makes the cursor appear when hovering between the "Send funds" button and the "QR div".